src/index.js

```diff
@@ -45,6 +45,23 @@ const saveTodoToLocalStorage = () => {
   localStorage.setItem('todos', JSON.stringify(todosArray));
 };
 
+// Add todo when enter key is pressed
+inputTodo.addEventListener('keypress', (e) => {
+  // Check if input is empty by removing the spaces
+  if (e.key === 'Enter' && inputTodo.value.trim() !== '') {
+    const todoValue = getValueFromInput();
+    todosArray.push({
+      index: todosArray.length + 1,
+      description: todoValue,
+      completed: false,
+    });
+    saveTodoToLocalStorage();
+    createTodo(todosArray[todosArray.length - 1]);
+    deleteTodo();
+  }
+} );
+
+
 const deleteTodo = () => {
```

Comment:
[REQUIRED] I am still experiencing the same problem mentioned by the previous reviewer. Whenever I click one of the bin icons, multiple to-do items get deleted.